src/server.cpp

```diff
@@ -2509,8 +2509,13 @@ void Server::fillMediaCache()
 
 			const std::string &filename = dln.name;
 			if (m_media.find(filename) != m_media.end()) // Do not override
```

Comment:
What about removing this `if` altogether and just ensuring media load order is right?

Reply:
If this latest commit isn't accepted I'll probably start finding this approach appealing